src/js.cookie.js

```diff
@@ -87,6 +87,16 @@
 					if (attributes[attributeName] === true) {
 						continue;
 					}
+
+					// Considers RFC 6265 section 5.2:
+					// ...
+					// 3.  If the remaining unparsed-attributes contains a %x3B (";")
+					//     character:
+					// Consume the characters of the unparsed-attributes up to,
+					// not including, the first %x3B (";") character.
+					// ...
+					attributes[attributeName] = attributes[attributeName].split(';')[0];
+
 					stringifiedAttributes += '=' + attributes[attributeName];
```

Comment:
We could write
```javascript
stringifiedAttributes += '=' + attributes[attributeName].split(';')[0];
```
and save some more bytes.

Reply:
<img width="841" alt="bildschirmfoto 2018-02-02 um 09 46 58" src="https://user-images.githubusercontent.com/21918/35724472-20ae8a66-07fe-11e8-9404-1696cce1b851.png">

Reply:
Well, unless `attributes[attributeName]` were needed afterwards for some other processing... but it doesn't look like it.

Reply:
Good idea! Done.